packages/banner/src/react/index.js

```diff
@@ -37,8 +44,21 @@ Banner.propTypes = {
 Banner.defaultProps = {
   color: vars.colors.blue
 }
-Banner.colors = vars.colors
 
+const Button = React.forwardRef((props, forwardRef) => (
```

Comment:
Kinda stinky that this forwardRef API from React requires that you explicitly write this at every custom level, instead of just showing up with props or something.